docs/examples/auto_scaling_cleanup.rb

```diff
@@ -2,10 +2,10 @@
 
 with_driver 'aws::eu-west-1' do
   aws_auto_scaling_group 'my-awesome-auto-scaling-group' do
-    action :delete
+    action :destroy
```

Comment:
:+1: for consistency (although this will break all existing cookbooks... again).  Can we just alias `:delete` to `:destroy` in the SuperLWRP so people have an easier migration?